types/react-i18next/src/I18nextProvider.d.ts

```diff
@@ -6,7 +6,7 @@ export interface I18nextProviderProps {
     i18n: i18n;
     initialI18nStore?: any;
     initialLanguage?: string;
-    children: JSX.Element;
+    children: React.ReactNode;
```

Comment:
Doe we even need to specify `children` here? They should be provided by the standard React typings, which means we could probably just completely remove them here.

Reply:
> They should be provided by the standard React typings

Actually standard React typing doesn't provide that. If we don't have `children` here, then `children`'s type is `never` so the user can't pass any children here.